src/integration-test/java/com/mongodb/jdbc/integration/testharness/DataLoader.java

```diff
@@ -54,7 +54,7 @@ public DataLoader(String dataDirectory) throws IOException {
         readDataFiles(dataDirectory);
     }
 
-    private void generateSchema() {
+    public void generateSchema() {
```

Comment:
Some changes to the dataloader class to load data and generate schema from the test case.